frontend/src/components/pages/Home/styles.module.scss

```
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css?family=Playfair+Display&display=swap"); //First font.
+@import url("https://fonts.googleapis.com/css?family=Roboto+Slab&display=swap"); //Second font.
+@import url("https://fonts.googleapis.com/css?family=Oswald&display=swap"); //Logo font.
+
+.background {
+  background-image: url("./img/Background\ image.jpg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-color: #172437;
+}
+
+//WELCOME SECTION//
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+.containerTitle {
```

Recuerda los espacios entre declaraciones dentro de todo este archivo.
```suggestion
}

.containerTitle {
```